README.md

```diff
@@ -103,7 +104,7 @@ Full customizable variables:
   --colorFocusInformation: #375de733;
   --colorInformationDark: #2c4ab8;
   --colorInformationDarker: #21388b;
-  --fontFamily: Roboto,'Helvetica Neue',Helvetica,Arial,sans-serif;
+  --fontFamily: roboto, 'Helvetica Neue', helvetica, arial, sans-serif;
```

Comment:
Why we change this? @tinhvqbk

Reply:
It seems like issues from stylelint ignore

Reply:
fixed on https://github.com/gotitinc/aha-css/commit/b5983bf953b651275b2a73511c6d4b231d8889d5